roles/packages/defaults/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+# defaults for radiorabe.common.package
+
+radiorabe_packages: [] # noqa: var-naming
```

Comment:
I believe the variable should be named `package_packages` for being compliant with the naming convention `<rolename>_<variablename>`.